File: Modules/HumanResource/MoneyClaimApplication/MoneyClaimReason/MoneyClaimReasonForm.cs
using Serenity.ComponentModel;
using System;

namespace HRMSoftware.MoneyClaimApplication.Forms;

[FormScript("MoneyClaimApplication.MoneyClaimReason")]
[BasedOnRow(typeof(MoneyClaimReasonRow), CheckNames = true)]
public class MoneyClaimReasonForm
{
    public string ClaimReason { get; set; }
    public string Description { get; set; }
}

File: Modules/ServerTypes/MoneyClaimApplication/MoneyClaimReasonForm.ts
import { StringEditor, PrefixedContext } from "@serenity-is/corelib";
import { initFormType } from "@serenity-is/corelib/q";

export interface MoneyClaimReasonForm {
    ClaimReason: StringEditor;
    Description: StringEditor;
}

export class MoneyClaimReasonForm extends PrefixedContext {
    static formKey = 'MoneyClaimApplication.MoneyClaimReason';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!MoneyClaimReasonForm.init)  {
            MoneyClaimReasonForm.init = true;

            var w0 = StringEditor;

            initFormType(MoneyClaimReasonForm, [
                'ClaimReason', w0,
                'Description', w0
            ]);
        }
    }
}